examples/browser/package.json

```diff
@@ -19,6 +19,15 @@
       }
     }
   },
+  "theiaPluginsDir": "../../plugins",
+  "theiaPlugins": {
+    "eclipse-theia.builtin-extension-pack": "https://open-vsx.org/api/eclipse-theia/builtin-extension-pack/1.88.1/file/eclipse-theia.builtin-extension-pack-1.88.1.vsix",
+    "ms-toolsai.jupyter": "https://open-vsx.org/api/ms-toolsai/jupyter/2024.6.0/file/ms-toolsai.jupyter-2024.6.0.vsix",
+    "ms-python.python": "https://open-vsx.org/api/ms-python/python/2024.12.3/file/ms-python.python-2024.12.3.vsix"
+  },
+  "theiaPluginsExcludeIds": [
+    "ms-python.vscode-pylance"
+  ],
```

Comment:
Suggestion: These should be put into the root package.json, which already includes the builtin extension pack.

Reply:
@msujew Sure I can move the two additional ones to the root. First I wanted to put them into the root, but I was not sure whether this change will affect the other non-playwright related jobs. And maybe people have some concerns having this extensions in the dev environment.

Reply:
It's IMO pretty unexpected to see the same plugin pack multiple times in the same repo. In that case, we shouldn't add `eclipse-theia.builtin-extension-pack` in this package.json, but use the `download:plugins` script from the root in addition to the one defined in the browser package.

Reply:
Moved to the root package.json